db/pqdb.go

```diff
@@ -0,0 +1,278 @@
+// Copyright ©2022 Servian
```

Comment:
db/pgdb.go ? or just db/postgres.go?

Reply:
(Yes, pq is the lib name but I'd prefer naming by the underlying database rather than implementation library we happen to be using.)

Reply:
I'm going to keep the short name.